pages/index.tsx

```diff
@@ -106,7 +109,11 @@ export default function Home({ deviceType, singleItemDetail }: Props) {
     dataFetcher,
     {
       isPaused: () => !coordinatesAndEventType,
-      onLoadingSlow: () => setSlowLoading(true),
+      onLoadingSlow: () => {
```

Comment:
@denizsult I understood the intention but I think here is a very late stage to show this modal. At this stage, user already loads the whole application and makes the request next. If they reach to that point without any issues I'd say they already have a device&connection that can make afetharita.com work. 

I believe we need something more like this https://stackoverflow.com/a/9011781 

What do you think?

Reply:
What he said may be true. Let me research a little more. Thanks a lot!

Reply:
@denizsult I am adding WIP label then, thank you!

Reply:
@cdagli  I implemented the solution we talked about.  Also, it should not be taken without the knowledge of the @fdemir.